README.md

```diff
@@ -106,14 +106,23 @@ Swift.
 The table below describes the current level of support that Swift Testing has
 for various platforms:
 
-| **Platform** | **Support Status** |
-|-|-|
-| Apple platforms | Supported |
-| Linux | Supported |
-| Windows | Supported |
-| FreeBSD, OpenBSD | Experimental |
-| Wasm | Experimental |
-| Android | Experimental |
+| **Platform**     | **Support Status** | **Qualification[^1]**        |
+| ---------------- | ------------------ | ---------------------------- |
+| Apple platforms  | Supported          | Automated CI: Build and Test |
+| Linux            | Supported          | Automated CI: Build and Test |
+| Windows          | Supported          | Automated CI: Build and Test |
+| FreeBSD, OpenBSD | Supported          | Manual                       |
+| Wasm             | Supported          | Automated CI: Build          |
```

Comment:
Wasm and Android aren't fully supported either. I'm not sure this table reflects reality right now. :/